cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -113,6 +113,11 @@ public Cube get() {
       // i.e. for flows only
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.FLOW)));
+    // queue
+    aggs.put("queue", new DefaultAggregation(
```

Comment:
I renamed it to flow.queue. Didn't know that matters. The other aggregation names don't have dots in them... But fine.